_data/sites.yml

```diff
@@ -88,6 +88,11 @@
   size: 7.33
   last_checked: 2022-04-21
 
+- domain: abridge.netlify.app
+  url: https://abridge.netlify.app/
+  size: 56
+  last_checked: 2022-08-22
```

Comment:
You seem to be back from the future. Do you mind using the current date?🙂

Reply:
Good eye, my brain got things backwards for a moment and put 22 for the day because I was thinking for whatever reason that it was also the year spot.